### ISSUE TITLE
The address is invalid error for orders-bulk.json

### ISSUE THREAD

**Issue by RecepCil** [codingexplained/complete-guide-to-elasticsearch#25]
I encountered an 'Invalid address' error while attempting to access the resources file in the **111. Introduction to Aggregations** lecture. Upon further investigation, I noticed that the file is not located within the Aggregations folder, which could be the root cause of the issue.

**Comment by ba0708** [comment]
Hi @RecepCil,

If I understand you correctly, you probably ran the command from within the `Aggregations` directory, while the command assumes that it is being run from the root directory. Try to either specify a full path to the file or run the command from the root directory and it should work. Sorry that this was not clear.

If this is not the cause, feel free to reopen the issue.

**Comment by RecepCil** [comment]
Hi @ba0708,

Let me rephrase myself. The command worked flawlessly on my local setup, and I truly appreciated the quality of the course—thank you for your dedication.

I meant that I had observed that the link you provided in the Udemy course for the "orders-bulk.json" file was broken.

<img width="353" alt="image" src="https://github.com/codingexplained/complete-guide-to-elasticsearch/assets/18148117/0888239c-59cf-4937-b9f0-3f1d457284ba">

Upon investigation, I found that the file is not located within the Aggregations folder. Although I initially thought this might be the cause of the issue, I noticed that you have already updated the link on the Udemy platform. Consequently, the link is now functional, and there is no longer a need to reopen the issue.

**Comment by ba0708** [comment]
Thank you for the clarification, @RecepCil, and thank you for the kind words!

That's a bit strange since that Udemy resource does not reference the GitHub repository in any way; the file has been uploaded directly to Udemy. I haven't changed anything, so if it didn't work at some point, it must have been a Udemy issue.
